### PR TITLE
chore(merge main): latest patched commit → e50c11b

### PR DESCRIPTION
**Pull request by Nurovek**
Error with tr elements in 
Orange-Boosted-Bootstrap\_gh_pages\docs\5.0\components\popovers\index.html
&
Orange-Boosted-Bootstrap\_gh_pages\docs\5.0\components\tooltips\index.html

### CODE REVIEW

File: site/content/docs/5.0/components/popovers.md [hunk 1]
```diff
@@ -269,12 +269,25 @@ Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` opt
       <td>null | function</td>
       <td><code>null</code></td>
       <td>Here you can supply your own sanitize function. This can be useful if you prefer to use a dedicated library to perform sanitization.</td>
+      <tr>
```
Comment (ffoodd) [inline site/content/docs/5.0/components/popovers.md:272]: There: you're opening a `<tr>` without closing the previous one…

Comment (Nurovek) [reply site/content/docs/5.0/components/popovers.md:272]: Thanks ! 🙏 Correcting right now

**Review by ffoodd — CHANGES_REQUESTED** [review]
Overall merge is really great!

You're only missing a few points here and there. A single point I didn't check is whether you thought about our custom examples that may need some love too (e.g. carousels not cycling) but I know we talked about it.

This is nicely done and almost perfect. :)

File: site/content/docs/5.0/components/popovers.md [hunk 2]
```diff
@@ -285,6 +298,18 @@ Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` opt
 Options for individual popovers can alternatively be specified through the use of data attributes, as explained above.
 {{< /callout >}}
 
+#### Using function with `popperConfig`
+
+```js
+var popover = new bootstrap.Popover(element, {
```
[the remark this answers is not in the document]
Comment (Nurovek) [reply site/content/docs/5.0/components/popovers.md:304]: Whoops, it seems I didn't care enough because it wasn't rejected, so I wasn't focused on it.

**Comment by ffoodd** [comment]
I guess you had enough fun with this, let's close it :D

**Pull request closed without merging**